### ISSUE TITLE
Keyboard navigation doesn't work in classes/guides tree

### ISSUE THREAD

**Issue by nene** [senchalabs/jsduck#235]
From Peter deHaan:

> I notice that in the TOC tree on the left, I can seemingly change the active node using the keyboard, but the right pane never loads the new node's content.
